### ISSUE TITLE
A11y - OU: Buttons must have discernible text

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5472]
Ensure buttons have discernible text. [https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|smart-link] 

h3. {{Element Location:}}

{noformat}#b2b-list-table-context-menu-button{noformat}

h3. To solve this problem, you need to fix at least (1) of the following:

* Element does not have inner text that is visible to screen readers
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element does not have an implicit (wrapped) <label>
* Element does not have an explicit <label>
* Element's default semantics were not overridden with role="none" or role="presentation"

!Screen Shot 2024-11-13 at 16.36.15 PM.png|width=1516,height=680,alt="Screen Shot 2024-11-13 at 16.36.15 PM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39573**



[created from NEXT-39573, comment 512394]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39573, comment 521037]
